### PR TITLE
DP-31678: Search and replace /media and /files URLs with /doc URLs in content

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-31678

**To Test:**
- [ ] run `drush msrm`

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by joeg8612** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9e9b83030d3bb05b679eb5ad30be624bf1fee748
</p>

File: docroot/modules/custom/mass_content/src/Commands/MassContentCommands.php [hunk 1]
```diff
@@ -456,4 +458,82 @@ public function getEntityBundles($type) {
...
+            }
+          }
+          if ($changed) {
+            if (method_exists($entity, 'setRevisionLogMessage')) {
+              $entity->setNewRevision();
+              $entity->setRevisionLogMessage('Revision created to fix raw media/ or files/ URL in the content.');
+              $entity->setRevisionCreationTime(\Drupal::time()
+                ->getRequestTime());
+            }
+            $entity->save();
```
Comment (tom-fleming) [inline docroot/modules/custom/mass_content/src/Commands/MassContentCommands.php:513]: @arthurbaghdas Just to sanity check, is there a chance that authors could receive watch notifications?

Comment (arthurbaghdas) [reply docroot/modules/custom/mass_content/src/Commands/MassContentCommands.php:513]: Forgot to add the snippet. thanks for reminding.

**Comment by tom-fleming** [comment]
Looks good to me. Leaving some notes in the ticket.

**Pull request merged**